README.md

```diff
@@ -1,18 +1,17 @@
-# Lottie for iOS, macOS (and [Android](https://github.com/airbnb/lottie-android) and [React Native](https://github.com/airbnb/lottie-react-native))
```

Comment:
Maybe you could link to the other platforms somewhere else (if we don't already do so)? I think being cross-platform is a key value prop of Lottie

Reply:
Good point, will make sure to include this somewhere